src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/LanguageServerTests.cs

```diff
@@ -1265,5 +1266,45 @@ public void TestInitialFixup()
             Assert.Equal(documentUri, response.Result.Uri);
             Assert.Equal("Price * Quantity", response.Result.Text);
         }
+
+        // https://github.com/microsoft/Power-Fx/issues/918
```

Comment:
no need to include the issue comment, imo

Reply:
Second thought, better without it.